packages/studio-plugin/tests/parsers/StaticParsingHelpers.test.ts

```diff
@@ -304,6 +304,22 @@ describe("parseExportAssignment", () => {
   });
 });
 
+it("parsing an negative numeric literal", () => {
+  const { sourceFile } = createTestSourceFile(`const a = { aKey: -1 }`);
```

Comment:
Small nit, but could this test instead parse a Component invocation with a negative prop value?